library/src/main/java/com/afollestad/date/controllers/DatePickerController.kt

```diff
@@ -106,7 +106,7 @@ internal class DatePickerController(
     if (notifyListeners) {
       notifyListeners(oldSelected) { calendar.clone() as Calendar }
     }
-    updateCurrentMonth(calendar)
+    updateCurrentMonth(calendar.clone() as Calendar)
```

Comment:
Since `calendar.clone() as Calendar` is also done on line 107 - it'd be better to pull this out to a `val` on line 105 and use that val in both of these places